ui/ozone/public/clipboard_data_bridge.h

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef UI_OZONE_PUBLIC_CLIPBOARD_DATA_BRIDGE_H_
+#define UI_OZONE_PUBLIC_CLIPBOARD_DATA_BRIDGE_H_
+
+#include <unordered_map>
+#include <string>
+#include <vector>
+
+#include "base/macros.h"
+#include "ui/ozone/ozone_base_export.h"
+
+namespace ui {
+
+using DataMap = std::unordered_map<std::string, std::vector<uint8_t>>;
```

Comment:
Do we really want to advertise this throughout the ui namespace? Maybe move it inside the ClipboardDataBridge?

Reply:
done